CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed the `AddToCart` button behavior in case of synchronization errors - @pkarw (#3150)
 - User token re-validation fixed to use proper HTTP codes - @pkarw (#3151, #3178)
 - Fixed undefined id of color swatches issue for simple product - @vishal-7037 (#3239)
+- Fixed product video embed on PDP - @juho-jaakkola (#3263)
```

Comment:
`1.10.0-rc.2` is in the correct place for this, right? Or should it also go into the next `1.9.x` release?

Reply:
1.10 is fine